docs/src/contributing/development-environment.md

```diff
@@ -207,25 +147,9 @@ If you are developing elsewhere, you must set up a registry yourself. Make sure
 !!! note
     If the development registry is insecure and has an IP address in a range other than `10.0.0.0/8`, `172.16.0.0/12`, or `192.168.0.0/16`, you'll have to modify `contrib/coreos/user-data.example` and whitelist your development registry so the daemons can pull your custom components.
 
-## Initial Platform Build
-
-The full environment is prepared. You can now build Deis from source code and then run the platform.
-
-We'll do three steps together:
-
-- Build the source (`make build`)
-- Update our local cluster with a dev release (`make dev-release`)
-- Start the platform (`deisctl start platform`)
-
-Conveniently, we can accomplish all three in one step:
-
-    $ make deploy
-
-Running `deisctl list` should display all of the services that your Deis cluster is currently running.
-
-You can now use your Deis cluster in all of the usual ways.
+## Initial Platform Installation
 
-At this point, you are running Deis from the code in your Git clone. But since rebuilding like this is time consuming, Deis has a simplified developer workflow more suited to daily development.
+TODO: Document helm install, or reference applicable existing documentation.
```

Comment:
[The charts repo's README](https://github.com/deis/charts/blob/master/README.md) should be sufficient